app/templates/base.html

```diff
@@ -19,7 +19,13 @@
     <script src="{{ url_for('static', filename='headscripts.js') }}" nonce="{{ csp_nonce() }}" ></script>
     <script nonce="{{ csp_nonce() }}">
         gtm_anon_id = '{{ gtm_anon_id }}';
+        window.sessionData = {{ {
+          "category_code": session.get('category', {}).get('code') | string,
+          "category_name": (session.get('category_answers', []) | first).get('answer_label') | string if session.get('category_answers') else None,
```

Comment:
Maybe this needs to be empty string if no value, incase it gets fed into an event so we can see it was unknown.  Please check for other values too.